localstack/utils/aws/aws_models.py

```diff
@@ -330,6 +340,18 @@ def get_qualifier_version(self, qualifier: str = None) -> str:
     def qualifier_exists(self, qualifier):
         return qualifier in self.aliases or qualifier in self.versions
 
+    @property
+    def envvars(self):
+        return self._envvars
+
+    @envvars.setter
+    def envvars(self, new_envvars):
+        for key, value in new_envvars.items():
```

Comment:
It would be nice if you can add a docstring here (we use reStructuredText usually) to document this condition.